app/src/androidTest/java/org/mozilla/reference/browser/ui/ContextMenusTest.kt

```diff
@@ -109,12 +109,12 @@ class ContextMenusTest {
     }
 
     @Test
-    fun textSelectionTest() {
+    fun copyTextTest() {
         val genericURL = TestAssetHelper.getGenericAsset(mockWebServer, 1)
 
         navigationToolbar {
         }.enterUrlAndEnterToBrowser(genericURL.url) {
-            longClickAndCopyPlainText("content")
+            longClickSelectAllAndCopyText(expectedText = "content", copy = true, selectAll = false)
```

Comment:
If `selectAll` is an optional parameter I don't think naming the function `longClickSelectAllAndCopyText` is best here because that would imply that's what the function does all the time. I prefer a variant of the original I think, `longClickCopyText`. I'm not sure why copy is a boolean parameter here too since that it what the function always does too. I would only pass in text and an optional `selectAll`.

Reply:
You're right, thanks for the heads-up !☺️ 
Made the changes and successfully re-ran 5x the test on Firebase.